src/main/java/teammates/common/datatransfer/questions/FeedbackMcqQuestionDetails.java

```diff
@@ -508,4 +468,27 @@ public String getQuestionTypeChoiceOption() {
     public String validateGiverRecipientVisibility(FeedbackQuestionAttributes feedbackQuestionAttributes) {
         return "";
     }
+
+    private Map<String, Integer> collateAnswerFrequency(List<FeedbackResponseAttributes> responses) {
+        Map<String, Integer> answerFrequency = new LinkedHashMap<>();
+
+        for (String option : mcqChoices) {
+            answerFrequency.put(option, 0);
+        }
+
+        if (otherEnabled) {
+            answerFrequency.put("Other", 0);
+        }
+
+        for (FeedbackResponseAttributes response : responses) {
+            String answerString = response.getResponseDetails().getAnswerString();
```

Comment:
This variable is used in a conditional branch that may not be reached. Let's not use a variable for it?